t/plugin/forward-auth.t

```diff
@@ -488,3 +488,127 @@ Authorization: 333
         }
     }
 --- error_code: 200
+
```

Comment:
Three blanks are needed between test cases

Reply:
> Three blanks are needed between test cases

Ok, I will fix that.